style.css

```diff
@@ -4,4 +4,171 @@
   height: 2766px;
   width: 1440px;
 }
-\n
\ No newline at end of file
+
+.header {
+  background-image: url(images/nav/pic-bgr.jpg);
+  height: 839px;
+  width: 1440px;
+}
+
+.nav {
+  display: flex;
+  margin-left: 585px;
+  width: 1440px;
+}
+
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6289680)